portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -44,6 +44,8 @@ const MNPLS_LNG = -93.2650;
 function onLoad() {
   loadSearch();
   renderLoginButton();
+  searchHash();
+  statisticsOnLoad();
```

Comment:
why did we add the statisticsOnLoad function here? initially i created that function specifically for the statistics.html page because it had to load extra elements that the other pages don't have to.

Reply:
+1, only concern is if we'll have other onLoad functions outside of what's on the statistics.html page

Reply:
I can put loadSearch( ) and searchHash( ) into statiscticsOnLoad( ).